ember-bootstrap/addon/components/bs-dropdown/menu.js

```diff
@@ -89,7 +90,18 @@ export default class DropdownMenu extends Component {
   }
 
   get alignClass() {
-    return this.align !== 'left' ? `dropdown-menu-${this.align}` : undefined;
+    if (macroCondition(getOwnConfig().isBS4)) {
+      return this.align !== 'left' ? `dropdown-menu-${this.align}` : undefined;
+    }
+
+    switch (this.align) {
+      case 'left':
+        return 'dropdown-menu-start';
```

Comment:
I double checked in Bootstrap docs for v5. If I get it right, `.dropdown-menu-start` is not needed as that's the default: https://getbootstrap.com/docs/5.3/components/dropdowns/#menu-alignment

Reply:
Ah, I had not noticed that. I will remove the left/start